src/bin/find.js

```diff
@@ -25,12 +25,20 @@ var getRuleURI = require('eslint-rule-documentation')
 
 var cli = require('../lib/cli-util')
 
-var processExitCode = argv.u && (!argv.n || argv.error) ? 1 : 0
 var getRuleFinder = require('../lib/rule-finder')
 var specifiedFile = argv._[0]
 
 var ruleFinder = getRuleFinder(specifiedFile, argv.core === false)
 
+var processExitCode
+// argv.error is true by default see line #20
+// if --no-error is passed argv.error will be false
+// if -n was passed it would set argv.error to false
+if (argv.n) {
+  argv.error = false
```

Comment:
you may prefer not to mutate `argv` here - ie, `const showError = argv.error && !argv.n;` and then `const processExitCode = argv.u && showError ? 1 : 0;`?